style.css

```diff
@@ -1,3 +1,84 @@
 #root {
-  color: red;
+  display: flex;
```

Comment:
I decided to go with css grid. Did you consider to use css grid for this project?